### PR TITLE
[WIP] Add more integration-tests

### PR DESCRIPTION
**Pull request by emilevauge**
This PR will add more integration tests.
It also replaces swarm with emilevauge/whoami.
More commits to come.
- [ ] Use https://github.com/ivpusic/httpcheck

### CODE REVIEW

File: integration/fixtures/docker/simple.toml [hunk 1]
```diff
@@ -9,8 +9,7 @@
 logLevel = "DEBUG"
 
 [docker]
-
 # It's dynamagic !
 endpoint = "{{.DockerHost}}"
-
 domain = "docker.localhost"
+watch = true
```
Comment (vdemeester) [inline integration/fixtures/docker/simple.toml]: @emilevauge I would prefer if we do a `simple-watch.toml` for the test that involve `watch` to be true (or the other way aronud). wdyt ? :stuck_out_tongue_closed_eyes:

Comment (emilevauge) [inline integration/fixtures/docker/simple.toml]: I agree :)

**Comment by emilevauge** [comment]
@rebaseTraefik

**Comment by rebaseTraefik** [comment]
I just pushed up the changes, enjoy!

**Comment by vdemeester** [comment]
Closing this for the time being…

**Pull request closed without merging**